.changes/v1.0.0/58-improvements.md

```diff
@@ -0,0 +1 @@
+*  Use the new v1alpha2 CCI types [GH-58]
```

Comment:
Instead of a new entry, I think we could modify `35-features.md` like this:
```
- **New Resource:** `vcfa_supervisor_namespace` to manage Supervisor Namespaces [GH-35, GH-58]
- **New Data Source:** `vcfa_supervisor_namespace` to read Supervisor Namespaces [GH-35, GH-58]
- **New Data Source:** `vcfa_kubeconfig` to get Kubeconfig [GH-35]
```

Reply:
Good suggestion, fixed.